src-java/nbworker-topology/nbworker-storm-topology/src/main/java/org/openkilda/wfm/topology/nbworker/services/FlowOperationsService.java

```diff
@@ -458,6 +461,20 @@ private boolean updateRequiredBySource(FlowPatch flowPatch, Flow flow) {
         return updateRequired;
     }
 
+    private boolean updateRequiredByMaxLatency(FlowPatch flowPatch, Flow flow) {
+        if (flowPatch.getMaxLatency() == null) {
+            return false;
+        }
+        return !Objects.equals(flowPatch.getMaxLatency(), flow.getMaxLatency());
+    }
+
+    private boolean updateRequiredByMaxLatencyTier2(FlowPatch flowPatch, Flow flow) {
+        if (flowPatch.getMaxLatencyTier2() == null) {
+            return false;
+        }
+        return !Objects.equals(flowPatch.getMaxLatencyTier2(), flow.getMaxLatencyTier2());
+    }
```

Comment:
I do not see where are you call these 2 methods

Reply:
These methods not used anymore, I moved the logic to `updateRequiredByPathComputationStrategy` and I removed usages, but not the methods themselves. Anyway, now these 2 methods are removed.